labs/Tempo.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Tempo
+parent: Labs
+---
+# Lab Name
+Tempo
+
+# Short Description
+Pure blockchain solution for time series data.
+
+# Scope of Lab
+The lab explores a non-intrusive method which is used to expand the storage and processing capabilities of the existing blockchain platform for time series data without damaging the decentralization feature of the blockchain, so that the blockchain can support more service scenarios, including IoT scenarios.
+
+The current blockchain platform or implementation has the following problems: Most blockchain platforms in the industry support only KV data storage, but not complex data storage. (relational data, streaming data, time series data, etc.); Currently, the smart contract of most blockchain platforms in the industry support only KV-type data storage and read interfaces, or enhanced query capabilities for pre-specified data types. (relational data, streaming data, time series data, etc.) OLTP (basic operation) and OLAP (advanced analysis) capabilities; If you need to support complex types of data processing and analysis capabilities, you need to intrusively modify the code of the blockchain platform, which is a very difficult development task and faces the problem of open source maintainence.
+
+The lab provide the solution to support timepoint data stored based on IoT time series database convergence wiht blockchain, thus make data can be written, read, aggregated, and analyzed through smart contract . For data aggregation and analysis tasks with a long time line and a large amount of data, parallel computing can be used to accelerate task execution based on the multi-point storage feature of blockchain. Supports latency-sensitive service scenarios (such as traffic governance). Finally, a pure on-link solution can be implemented for IoT time series data, which does not depend on off-link components. Data storage, aggregation, computing, and query can be completely decentralized. It is not bound to a blockchain platform or time series database. You can use plug-ins to flexibly expand the application scenarios (such as Hyperledger Fabric and GaussDB).
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/cloud1230
+- https://github.com/LilkkJerry
+- https://github.com/KaiQiu9527
+
+# Sponsor
+
+
+# Pre-existing repository
+https://github.com/HWBlockchainLab/tempo
```

Comment:
In order to bring an existing source base to the Hyperledger Foundation, the repository must have [DCO](https://developercertificate.org/) signoff on all commits. Since your existing repository does not have DCO signoff on all commits, you will need to do one of two things:

1. bring your code by squashing all of your commits into a single first commit made against a new lab repo with your DCO sign-off.

2. amend the commit history to include DCO sign-off for each of the commits. The Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). Also, the [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.

Please let us know which option you would like to take for this repository.

Reply:
Thanks tracy for giving suggestions, we are handling the DCO issues.

Reply:
Sorry for reply late. We re-submit and sign all the commits. Please help check. https://github.com/HWBlockchainLab/tempo/tree/dev

Reply:
All branches and all commits will require DCO sign-off before we can bring this code into the Hyperledger Labs organization. It might be easier to use option 1 and squash all commits into a single commit within the new lab repo. For me, this is the only thing holding up my approval of this as a lab.

Reply:
Thanks for the update, I have deleted the old branch and leave the dco sign-off one. Please help check. 
Thanks again.